packages/apis/src/verifier_simple.rs

```diff
@@ -95,6 +95,29 @@ pub struct OperatorVoteInfoResponse {
     pub result: String,
 }
 
+/// Metadata for a task - indexed by (task_queue, task_id)
+#[cw_serde]
+pub struct TaskMetadata {
```

Comment:
Happy to make this common, let's add a `pub struct Tasks<'a>(Map<'a, (&'a, Addr, TaskId), TaskMetadata>)` as well as in  cw-controllers

Reply:
I think b9c3011c0157 changes this one